package.json

```diff
@@ -38,6 +38,7 @@
     "faker": "^5.5.3",
     "form-data": "^4.0.0",
     "generate-password": "^1.6.0",
+    "lighthouse": "^9.6.1",
```

Comment:
are these libs used anywhere? `lighthouse ` `playwright-lighthouse ` `request`

Reply:
No, just forgot to remove those.